### ISSUE TITLE
UI feature requests?

### ISSUE THREAD

**Issue by mjm716** [jacquesh/foo_openlyrics#57]
-Ability to edit the default display when no lyrics are present?
-Ability to (mouse-wheel) scroll lyrics?

**Comment by jacquesh** [comment]
Yeah editing the default display is a thing I could do probably. What would you set it to be?

You can already mouse-wheel scroll if you set scrolling to manual (in the display settings). I'm unlikely to add the ability to scroll manually when automatic scrolling is enabled.

**Comment by mjm716** [comment]
Optimal would be default display field to be customized by titleformat code.

Thanks for manual tip. 
Perhaps (if synced lyric and) if manual is set, there is a ctrl-click option for "sync to playing" so user can quickly get to accurate playing point.

**Comment by jacquesh** [comment]
Yes indeed, a customisable titleformat string is what I would make it if I added the option.

My question is: What would *you* set it to, if not the existing default? :)

**Comment by mjm716** [comment]
Not saying that full titleformatting code is necessary - simple text, tags and linebreaks could be enough - but I'm partial to info overload and have obsessively complete tags, so just a _lot_ more. ;)

`$if(%isplaying%,,Play some music.)
>>>[%artist% _ ][%title%][$replace($max(%play_count%,%lastfm_play_count%),1,$char(185),2,$char(178),3,$char(179),4,$char(8308),5,$char(8309),6,$char(8310),7,$char(8311),8,$char(8312),9,$char(8313),0,$char(8304))] $repeat(★,%rating%)<<<$crlf()
<<['('[%DISCOGS_TRACK_CREDITS% ][feat. %DISCOGS_CREDIT_FEATURING% ][%performer% ][vox. %DISCOGS_CREDIT_VOCALS% ][%DISCOGS_CREDIT_REMIXED_BY% Remix]')']>>$crlf()

$if(%DISCOGS_ARTIST_PROFILE%,%artist%: %DISCOGS_ARTIST_PROFILE%$crlf()' '$crlf(),)
>>>[%album%][ '('%album_format%')'][, $if2(%releaseyear%,%date%)][+%RELEASE DATE%]<<<$crlf()
<[%publisher% ][#%DISCOGS_CATALOG% ][%DISCOGS_COUNTRY%]>$crlf()
<[/[%key%][_%BPM%][_%bitrate%k] _ %genre%]>$crlf()
<<<[Added: $left(%added%,7) $char(185)%first_played%]>>>$crlf()
[' '$crlf()<<'Credits:' %DISCOGS_RELEASE_CREDITS%>>]$crlf()
' '$crlf()
['Release notes:' %DISCOGS_RELEASE_NOTES%]$crlf()
[%comment%]$crlf()`

**Comment by FaceCrap** [comment]
Surprised that this is still not a thing. It's one of the reasons I still won't use it.

> My question is: What would you set it to, if not the existing default? :)

To begin with, full titleformatting support would be a pre-req.

![image](https://github.com/jacquesh/foo_openlyrics/assets/95778718/d204b793-f9e1-46f4-beca-380ec46c4fd7)
